master/buildbot/test/unit/test_db_users.py

```diff
@@ -149,13 +150,13 @@ def thd(conn):
                     sorted([tuple(i) for i in infos])
                 ), (
                     [
-                        (1L, u'soap', None, None),
-                        (2L, u'lye', None, None),
-                        (3L, u'marla', u'marla', u'cancer'),
+                        (int(1), u'soap', None, None),
+                        (int(2), u'lye', None, None),
+                        (int(3), u'marla', u'marla', u'cancer'),
                     ], [
-                        (1L, u'IPv9', u'0578cc6.8db024'),
-                        (2L, u'git', u'Tyler Durden <tyler@mayhem.net>'),
-                        (2L, u'irc', u'durden')
+                        (int(1), u'IPv9', u'0578cc6.8db024'),
+                        (int(2), u'git', u'Tyler Durden <tyler@mayhem.net>'),
+                        (int(2), u'irc', u'durden')
```

Comment:
I agree. It is odd. If there are no particular reason those numbers are long, I would prefer to just have it be "2".